internal/doctl/mylib.go

```diff
@@ -0,0 +1,182 @@
+package mylib
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/digitalocean/godo"
+	"github.com/pkg/errors"
+)
+
+//DoctlDependencies interface for doctl dependent functions
+type DoctlDependencies interface {
```

Comment:
interfaces should be implemented where they are used. This interface should be in the `main.go`.

Reply:
struct was not able to import interface from main